### PR TITLE
List component, Input component refactor

### PR DESCRIPTION
**Pull request by bharati-21**
- Refactored list component:
  - Added new list component: inline-list
  - Removed padding, margin from styles and added as util classes in HTML code
  - Moved list component from docs/index.html to docs/list/index.html

- Refactored list component:
   - Added new input component : inline-input
   - Removed padding, margin from styles and added as util classes in HTML code
   - Moved input component from docs/index.html to docs/input/index.html

### CODE REVIEW

**Review by rishbitsnbytes — COMMENTED** [review]
Everything looks good to me.

File: Components/input/input.css [hunk 1]
```diff
@@ -2,27 +2,56 @@
...
-    -ms-border-radius: var(--br-5);
-    -o-border-radius: var(--br-5);
+    border-radius: var(--br-4);
+    -webkit-border-radius: var(--br-4);
+    -moz-border-radius: var(--br-4);
+    -ms-border-radius: var(--br-4);
+    -o-border-radius: var(--br-4);
     border: 1px solid var(--text-color);
+
     background-color: #fff;
```
Comment (ravikumar0403) [inline Components/input/input.css:23]: I think you missed using variable here for this color

File: Components/input/input.css [hunk 2]
```diff
@@ -2,27 +2,56 @@
...
 
+.input-inline .input-text {
+    border: none;
+    background-color: transparent;
+    outline: none;
+}
+
+.input-inline .text-label {
+    background-color: #ffffff;
+    border-radius: var(--br-4);
```
Comment (ravikumar0403) [inline Components/input/input.css:38]: here as well, use variable for colors

**Review by ravikumar0403 — APPROVED** [review]
Good for now, but can improve the class names with a consistent naming convention.

I see you have tried using BEM naming conventions for some class names whereas in other places it's just hyphenated class names and in some places, there are just double hyphenated classes which I assume you used as Element--Modifier. This may not be the best experience for a user using your library.

**Comment by bharati-21** [comment]
> Good for now, but can improve the class names with a consistent naming convention.
> 
> I see you have tried using BEM naming conventions for some class names whereas in other places it's just hyphenated class names and in some places, there are just double hyphenated classes which I assume you used as Element--Modifier. This may not be the best experience for a user using your library.

Thank you for your review.
I used BEM only for the documentation website containers and elements. For the components in the library, I used only OOCSS.

File: Components/input/input.css (hunk 1 above)
Comment (bharati-21) [reply Components/input/input.css:23]: Thank you for pointing this out. I shall change it 👍🏽

**Comment by bharati-21** [comment]
> Everything looks good to me.

Thank you for your review.

**Pull request merged**